profiles/schema.py

```diff
@@ -0,0 +1,74 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+
+
+class DivisionOfInterestType(DjangoObjectType):
+    class Meta:
+        model = DivisionOfInterest
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+
+    concepts_of_interest = graphene.List(ConceptType)
+    divisions_of_interest = graphene.List(AdministrativeDivisionType)
+
+
+Language = graphene.Enum("Language", [(l[1], l[0]) for l in settings.LANGUAGES])
+ContactMethod = graphene.Enum(
+    "ContactMethod", [(l[1], l[0]) for l in settings.CONTACT_METHODS]
```

Comment:
well, `l` represented `language` 😅 so, maybe `cm` in this case

Reply:
I would actually enforce uppercase on the lable, to be more graphql compliant.
```python
ContactMethod = graphene.Enum(
    "ContactMethod", [(cm[1].uppercase(), cm[0]) for cm in settings.CONTACT_METHODS]
)
```

Reply:
Copy paste :)